.github/workflows/mint-nc-tests.yaml

```diff
@@ -0,0 +1,49 @@
+
+name: Mint NC Tests
+on: [workflow_call]
+
+jobs:
+  mint-nc-tests:
+    name: Mint NC Tests
+    runs-on: ubuntu-latest
+    timeout-minutes: 90
+    permissions:
+      actions: read         # download-artifact
+      contents: read        # required for actions/checkout
+    steps:
+      - name: Checkout noobaa-core
+        uses: actions/checkout@v4
+        with:
+          repository: 'noobaa/noobaa-core'
+          path: 'noobaa-core'
+
+      - name: Download artifact
+        uses: actions/download-artifact@v4
+        with:
+          name: noobaa-tester
+          path: /tmp
+
+      - name: Load image
+        run: docker load --input /tmp/noobaa-tester.tar
+
+      - name: Create Mint logs directory
+        run: |
+          set -x
+          cd ./noobaa-core
+          mkdir -p logs/mint-nc-test-logs
+          chmod 777 logs/mint-nc-test-logs
+
+      - name: Run NC Mint tests
+        run: |
+          set -x
+          cd ./noobaa-core
+          make test-nc-mint -o tester
+
+      - name: Upload artifact
+        if: always()
+        uses: actions/upload-artifact@v4
+        with:
+          name: mint-nc-test-logs
```

Comment:
what is this artifact used for?

Reply:
for debugging, as we agreed I removed the artifacts for now and added local debugging docs so people will understand how to debug it if a problem arises